packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -356,7 +355,7 @@ def _check_diagnostics_bundle_status(dcos_api_session):
             assert required_status_field in properties, 'property {} not found'.format(required_status_field)
 
 
-def _create_bundle(diagnostics: Diagnostics):
+def _create_bundle(diagnostics: Diagnostics) -> Any:
```

Comment:
The return type seems to be `str`.

Reply:
Force to use `str`, I have use Any because we do not have exact type of `create_response['extra']['bundle_name']`